packages/client/hmi-client/src/services/project.ts

```diff
@@ -154,16 +154,16 @@ async function getPublicationAssets(projectId: string): Promise<ExternalPublicat
  * @assetId string - represents the id of the asset to be added. This will be the internal id of some asset stored in one of the data service collections
  * @return any|null - some result if success, or null if none returned by API
  */
-async function addAsset(projectId: string, assetsType: string, assetId: string) {
+async function addAsset(projectId: string, assetType: string, assetId: string) {
```

Comment:
This function could be a good place to add the logger informing the user that the asset has been added to the project or not. Especially that the `assetType` is a string and therefore easy to display.

Reply:
I didn't do that because I'd have to pass the `assetName` and `projectName` which are different